<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Uimessages</class>
 <widget class="QMainWindow" name="Uimessages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Direct messages</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QPushButton" name="pushButtonFermer">
      <property name="text">
       <string>Fermer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="userList"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="messageLayout">
      <item>
       <widget class="QTextEdit" name="messageDisplay">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="inputLayout">
        <item>
         <widget class="QLineEdit" name="messageInput"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSend">
          <property name="text">
           <string>Envoyer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
